SU-SWS/stanford_fwb
# BASIC-2369 Renamed view & created update path

# NOT READY

# Summary
- Renamed the block to something more appropriate
- Created update path to update tables `block`, `context`, `views_display`, & any block reference fields.

# Needed By (Date)
- Soon

# Urgency
- High urgency

# Steps to Test

1. before checking out the branch, clone repo, enable the module
2. Check out the `peec-view-modes` branch of `stanford_image_styles`
3. Create a "Stanford Slide" node
4. place the "midget" block in the "Full Width Top" region to view the rendered output
5. Check out the `BASIC-2369-rename` branch of `stanford_fwb` and the `BASIC-2369` branch of `stanford_image_styles`
6. Run database updates.
7. verify block still exists in its configured location.

# Affected sites:
ds_casbs
ds_ethics
ds_fc-planning
ds_hsdo
ds_jsa-peec
ds_jumpstart-dreme
ds_jumpstart-graphicnovel
ds_jumpstart-neuro
ds_jumpstart-vptlplus
ds_lbre-dev
ds_lbre-jumpstart
ds_lbre-preview
ds_lbre-production
ds_mlk
ds_mlk-dev
ds_peec-sandbox
ds_sa-studentservices
ds_tedx
ds_undergrad
ds_undergrad-dev

sites where stanford_fwb-7.x-1.0 is not installed but in code:
ds_grand-tour
ds_heidegger
ds_religious-studies
ds_sws-build-jsa
ds_sws-uat-jsa
